controllers/secretsync/secret_sync.go

```diff
@@ -74,7 +77,7 @@ func (r *SecretReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 			&corev1.Secret{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      request.Name,
-					Namespace: request.Namespace,
+					Namespace: r.TargetNamespace,
```

Comment:
Since this replacement is happening, shouldn't this also log using `r.TargetNamespace` rather than `request.Namespace`?

Reply:
I kept the log the same because it's logging the object that triggered the reconcile, which I think is what we want, but I am open to other ideas.

Reply:
Ah, I see. Hmmm. Maybe add it in addition to the others?

Reply:
I just added it.

Reply:
For the `secret_sync` also?

Reply:
Sorry, I added that one too.